src/main/java/org/minions/devfund/chelo/WhackAMole.java

```diff
@@ -0,0 +1,242 @@
+        return moleGrid[x][y];
+    }
+
+    /**\
+     * A method to return score value.
+     * @return score
+     */
+    public int getScore() {
+        return score;
+    }
+
+    /**
+     * A method to return moles left value.
+     * @return remaining moles
+     */
+    public int getMolesLeft() {
+        return  molesLeft;
+    }
+
+    /**
+     * A method to print the current status for.
+     * Score,
+     * Attempts Left,
+     * Moles Left.
+     */
+    public void printStatus() {
+        System.out.println("Score: " + this.score
+                + ", Attempts left: " + this.attemptsLeft
+                + ", Moles left: " + this.molesLeft);
+    }
+
+    /**
+     * Main method to run the program.
+     * @param args main method
+     */
+    public static void main(final String[] args) {
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        System.out.println("Enter number of attempts:");
+        int numAttempts = scanner.nextInt();
+        System.out.println("Enter dimension for the grid:");
+        int dimensionBegin = scanner.nextInt();
+
+        WhackAMole whackAmole = new WhackAMole(numAttempts, dimensionBegin);
+
+        System.out.println("Enter the amount of moles");
+        int amountMoles = scanner.nextInt();
+        whackAmole.placeTheMoles(amountMoles, dimensionBegin);
+
+        // Printing the grid as first action to know where the moles were placed
+        System.out.println(whackAmole.printGrid());
+
+        int userInputX;
+        int userInputY;
+
+        // While the attempts left are greater than zero the user can submit coordinates
+        while (numAttempts > 0) {
+            if (whackAmole.getMolesLeft() == 0) {
+                System.out.println("Whack-A-Mole CHAMP!!!");
+                break;
+            }
+
+            // Reading user input
+            System.out.println("Enter X: ");
+            userInputX = scanner.nextInt();
+            System.out.println("Enter Y: ");
+            userInputY = scanner.nextInt();
+
+            // Printing the coordinates submitted by the user
+            System.out.println("User input: [X=" + userInputX + "][Y=" + userInputY + "]");
+
+            // Conditional if the user wants to finish the game
+            if (userInputX == -1 || userInputY == -1) {
+                System.out.println(whackAmole.printGrid());
+                whackAmole.printStatus();
+                System.out.println("GAME OVER: User gave up");
+                numAttempts = 0;
+            } else {
+                // Positioning the user's hit
+                whackAmole.whack(userInputX, userInputY);
+                System.out.println(whackAmole.printGridToUser());
+                whackAmole.printStatus();
+                numAttempts--;
+
+                if (numAttempts == 0) {
+                    System.out.println(whackAmole.printGrid());
+                    System.out.println("GAME OVER! NO MORE ATTEMPTS");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)